Lib/slapdtest/_slapdtest.py

```diff
@@ -279,7 +279,6 @@ def _cleanup_rundir(self):
         """
         # cleanup_rundir() is called in atexit handler. Until Python 3.4,
         # the rest of the world is already destroyed.
-        import os, os.path
```

Comment:
Don't remove the local import here. The requirement is documented in the comment above the import.

Reply:
Thanks for the review.

I'm curious, do you have any documentation, resource, or experiment that demonstrates why this import and comment is necessary?

I did experimentation with Fedora's Python 2.7.14. To experiment, I applied the following diff to force all cleanup during the atexit handler:

```patch
diff --git a/Lib/slapdtest/_slapdtest.py b/Lib/slapdtest/_slapdtest.py
index bb03a1f..f38bf6e 100644
--- a/Lib/slapdtest/_slapdtest.py
+++ b/Lib/slapdtest/_slapdtest.py
@@ -441,6 +441,7 @@ class SlapdObject(object):
         """
         Stops the slapd server, and waits for it to terminate and cleans up
         """
+        print('stop')
         if self._proc is not None:
             self._log.debug('stopping slapd with pid %d', self._proc.pid)
             self._proc.terminate()
@@ -575,6 +576,6 @@ class SlapdTestCase(unittest.TestCase):
         cls.server = cls.server_class()
         cls.server.start()
 
-    @classmethod
-    def tearDownClass(cls):
-        cls.server.stop()
+    # @classmethod
+    # def tearDownClass(cls):
+    #     cls.server.stop()
```

The atexit handler never had a problem or failure using the module level `os` import. From what I can tell, this comment seems unnecessary or outdated. 

How can I demonstrate to myself this import is necessary?